packages/react-core/src/components/ToggleGroup/examples/ToggleGroupCompact.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+
+export const ToggleGroupCompact: React.FunctionComponent = () => {
+  const [state, setState] = React.useState({
+    isSelected: {
+      'compact-1': false,
+      'compact-2': false
+    }
+  });
```

Comment:
This applies to all of the example files, but rather than using "state" and "setState", which aren't entirely descriptive, I'd suggest instead using "isSelected" for something more along the lines of:

```suggestion
  const [isSelected, setIsSelected] = React.useState({
    'toggle-group-compact-1': false,
    'toggle-group-compact-2': false
  });
```

It may not be entirely important considering the context of the example is fairly limited, but I think it'd be better to provide descriptive variable names for those reading/using the example code.